tools/python/boutiques/searcher.py

```diff
@@ -13,7 +13,7 @@ class Searcher():
 
     def __init__(self, query, verbose=False, sandbox=False, max_results=None,
                  no_trunc=False, exact_match=False):
-        if len(query or '') > 0:
+        if query is not None:
```

Comment:
Not sure why this is reverted: it was implemented in https://github.com/boutiques/boutiques/pull/534/files